core/src/i18n/en.yaml

```diff
@@ -1115,13 +1115,15 @@ namespaces:
 no-permissions:
   title: No Permissions
 node-details:
+  all: All Messages
```

Comment:
Tbh instead of (all messages, normal messages, warnings) I'd go with (all, info, warnings), but it's just my idea, let's see wait for TWs opinion.